app/views/hyrax/admin/stats/_downloads.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>Total Downloads</h2>
+<ul>
+  <%= @total_downloads %>
+</ul>
+<br/>
+<h2>Downloads per Work</h2>
+<ul>
+  <% @works_to_display.each do |dwnld| %>
+    <li><%= dwnld.title %></li><span class="count">(<%= dwnld.downloads %>)</span>
```

Comment:
This markup seems odd. I'm not sure `ul` can have any children besides `li`. Did you want to tuck the `span` within the `li`?

Reply:
Yes `span` within `li` is nicer thanks. I'll see how else we may want this styled.